ledger-core/insolar/payload/validation.go

```diff
@@ -177,3 +179,44 @@ func (m *VStateReport) validateUnimplemented() error {
 
 	return nil
 }
+
+func (m *VFindCallRequest) Validate(currPulse PulseNumber) error {
+	var (
+		lookAtPulse   = m.GetLookAt()
+		calleePulse   = m.GetCallee().GetLocal().Pulse()
+		outgoingPulse = m.GetOutgoing().GetLocal().Pulse()
+	)
+
+	switch {
+	case lookAtPulse < pulse.LocalRelative:
```

Comment:
I've found out that there's helper function on PulseNumber: `IsValidAsPulseNumber`

Reply:
Done